test/e2e/templates/column.tpl.html

```diff
@@ -42,4 +52,4 @@
     {x: 7, val_0: 4.927, val_1: 3.35, val_2: -13.074, val_3: -12.625}
   ]
 }
-{% endblock %}
```

Comment:
Sorry my question looks weird in this context; it seems you are automatically appending line-breaks to the end of files where there is none - which is nice. So I thought you have a plugin that does this automatically :) (so I can install it also)